src/test/java/edu/hm/hafner/util/SetTest.java

```diff
@@ -0,0 +1,198 @@
+package edu.hm.hafner.util;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+
+/**
+ * Test the class {@link java.util.HashSet} for integers.
+ *
+ * @author Johannes Reichle
+ */
+class SetTest {
+
+    // methods usable that are not tested yet???
+    @Test
+    void shouldCreateEmptySet() {
+        HashSet<Integer> testSet = creatHashSet(0);
+
+        Assertions.assertThat(testSet).isNotNull();
```

Comment:
```suggestion
        assertThat(testSet).isNotNull();
```
Und statischen Import nutzen